src/plays/cryptocurrency-tracker/cryptocurrencyTracker.css

```diff
@@ -0,0 +1,51 @@
+/* enter stlyes here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+* {
```

Comment:
Do not override selectors. Have a class at your play level and apply styles. Otherwise it may impact something else at the app level.